File: 18_simple_bank_system/python/simple_bank_system.py
balance = 0.0

while True:
    print("\n1. Check Balance")
    print("2. Deposit")
    print("3. Withdraw")
    print("4. Exit")

    choice = input("Choice: ")

    if choice == "1":
        print("Balance:", balance)

    elif choice == "2":
        amount = float(input("Enter amount: "))
        balance += amount
        print("Amount deposited.")

    elif choice == "3":
        amount = float(input("Enter amount: "))
        if amount <= balance:
            balance -= amount
            print("Amount withdrawn.")
        else:
            print("Insufficient balance.")

    elif choice == "4":
        break

    else:
        print("Invalid choice.")

File: 18_simple_bank_system/java/SimpleBankSystem.java
import java.util.Scanner;

public class SimpleBankSystem {
    public static void main(String[] args) {

        double balance = 0.0;
        Scanner sc = new Scanner(System.in);

        while (true) {
            System.out.println("\n1. Check Balance");
            System.out.println("2. Deposit");
            System.out.println("3. Withdraw");
            System.out.println("4. Exit");
            System.out.print("Choice: ");

            String choice = sc.nextLine();

            switch (choice) {
                case "1":
                    System.out.println("Balance: " + balance);
                    break;

                case "2":
                    System.out.print("Enter amount: ");
                    balance += Double.parseDouble(sc.nextLine());
                    System.out.println("Amount deposited.");
                    break;

                case "3":
                    System.out.print("Enter amount: ");
                    double amount = Double.parseDouble(sc.nextLine());
                    if (amount <= balance) {
                        balance -= amount;
                        System.out.println("Amount withdrawn.");
                    } else {
                        System.out.println("Insufficient balance.");
                    }
                    break;

                case "4":
                    sc.close();
                    return;

                default:
                    System.out.println("Invalid choice.");
            }
        }
    }
}

